advanced/mathfx/src/main/java/org/arakhne/afc/math/geometry/d2/dfx/MultiShape2dfx.java

```diff
@@ -235,6 +235,16 @@ public void invalidated(Observable observable) {
 			}
 		}
 
+		public boolean equals(Object obj) {
+			if (! super.equals(obj)) {
+				return false;
+			}
+			InternalObservableList lObj = (InternalObservableList) obj;
+			if (internalList.equals(lObj.internalList)) {
+				return true;
+			}
+			return false;
+		}
 	}
```

Comment:
The implementation of the `hashCode()` is missed. When a `equals(Object)` function is coded, the `hashCode()` function must be too.